include/sir/platform.h

```diff
@@ -163,7 +163,21 @@ int pthread_getname_np(pthread_t thread, char* buffer, size_t length);
 #    if !defined(_GNU_SOURCE)
 #     define _GNU_SOURCE 1
 #    endif
-#    if defined(__GLIBC__) && __GLIBC_MINOR__ >= 12
+#    if defined(__has_include)
+#     if __has_include(<features.h>)
+#      include <features.h>
+#     endif
+#    endif
+#    if defined(__GLIBC__)
+#     if defined(GLIBC_VERSION)
+#      undef GLIBC_VERSION
```

Comment:
Can't you just do undef? Same end result, less platform.h, which is kind of gnarly by now.